ui/scripts/movie.js

```diff
@@ -109,6 +123,31 @@ export default class Movie {
         return conteiner;
     }
 
+    // expects countries object that has name and flag fields, class  
+    getFlags(counries, name) {
+        let flags = "";
+
+        if (counries.length !== 0) {
+            const list = document.createElement("ul");
+            list.classList.add(name);
+
+            counries.map(country => {
+                if (country.flag) {
```

Comment:
nit: you access `country.flag` twice: here and in the next line. It's usually better to avoid repeating the work by extracting a value once. E.g.:

```
flag = country.flag;
if (flag) {
  flags += `...{flag}...`
}
```

(but see the other suggestion below that makes this comment not needed)